e2e-test/src/tests/cleanUp.test.js

```diff
@@ -0,0 +1,106 @@
+
+    const fileInputElementHandle = await page.waitForXPath("//input[@type='file']");
+    await fileInputElementHandle.uploadFile('../e2e-test/data/cleanUpTest.csv');
+    await page.waitForXPath(vListTile({ title: 'cleanUpTest.csv ', content: 'Dataset processed with 1 validation failures' }));
+
+    await expect(page).toClickXPath(vListTile({ title: 'cleanUpTest.csv ', content: 'Dataset processed with 1 validation failures' }) + vBtn('View Data'));
+
+    //  verify the dataset is failing
+    await page.waitForXPath(vListTile({ title: 'cleanUpTest.csv' }) + vIcon({ cssClass: 'mdi mdi-alert theme--light warning--text' }));
+
+    // select the third column(column 'C') and mark it as a 'Group'
+    await expect(page).toClickXPath("//div[@class='column-header-cell type-measurement'][contains(text(), 'C')]");
+    await expect(page).toClickXPath("//input[@value='group']");
+
+    //  verify the dataset is ready to process but needs to be more explict
+    await page.waitForXPath(vListTile({ title: 'cleanUpTest.csv' }) + vIcon({ cssClass: 'mdi mdi-check theme--light success--text' }));
+  });
+
+  it('mask all and unmask', async () => {
+    await Promise.all([
+      page.goto(CLIENT_URL),
+      page.waitForNavigation({ waitUntil: 'networkidle0' }),
+    ]);
+    await expect(page).toClickXPath(vBtn('Your Datasets'));
+
+    const fileInputElementHandle = await page.waitForXPath("//input[@type='file']");
+    await fileInputElementHandle.uploadFile('../e2e-test/data/cleanUpTest1.csv');
+    await page.waitForXPath(vListTile({ title: 'cleanUpTest1.csv ', content: 'Dataset processed with 1 validation failures' }));
+
+    await expect(page).toClickXPath(vListTile({ title: 'cleanUpTest1.csv ', content: 'Dataset processed with 1 validation failures' }) + vBtn('View Data'));
+
+    //  verify the dataset is failing
+    await page.waitForXPath(vListTile({ title: 'cleanUpTest1.csv' }) + vIcon({ cssClass: 'mdi mdi-alert theme--light warning--text' }));
+
+    // verify the dataset gets an error
+    await expect(page).toClickXPath(vListTile({ title: 'Non-numeric column' }));
+
+    // click 'mask all' to fix it
+    await expect(page).toClickXPath(vBtn('Mask all'));
+
+    //  verify the dataset is ready to process again
+    await page.waitForXPath(vListTile({ title: 'cleanUpTest1.csv' }) + vIcon({ cssClass: 'mdi mdi-check theme--light success--text' }));
+
+    // verify the column is masked and then click on the masked column
+    await expect(page).toClickXPath("//div[@class='column-header-cell type-masked mdi mdi-eye-off']");
+
+    // select the masked column and mark it as a 'Group'
+    await expect(page).toClickXPath("//input[@value='group']");
+
+    //  verify the dataset is ready to process again
+    await page.waitForXPath(vListTile({ title: 'cleanUpTest1.csv' }) + vIcon({ cssClass: 'mdi mdi-check theme--light success--text' }));
+  });
+
+  it('fix selecting wrong data type for file name', async () => {
```

Comment:
There might be a better way to name this

Reply:
`mark as metabolite + metadata`?

Reply:
By the way, while it's generally a good idea to have separate tests for separate functions, this is a case where it's just so much easier to test the `Metabolite` and `Metadata` buttons in the same test.